### PR TITLE
Use slick in at least a rudimentary way

### PR DESCRIPTION
**Pull request by nafg**
The `java.sql.Connection` is now attained from Slick, not from play's API.

**\* IMPORTANT: You must modify your `application.conf`! See `application.conf.sample` ***

### CODE REVIEW

**Comment by nafg** [comment]
Review by @yankiw

File: app/AppLoader.scala [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import controllers.Assets
+import play.api.ApplicationLoader.Context
+import play.api._
+import db.slick.{DbName, SlickComponents}
+import router.Routes
+import slick.driver.PostgresDriver
+
+class AppLoader extends ApplicationLoader {
+  def load(context: Context) = new MyComponents(context).application
```
Comment (yankiw) [inline app/AppLoader.scala]: can you wrap the new object in paranthesis please

Comment (nafg) [inline app/AppLoader.scala]: Done

**Comment by nafg** [comment]
Needs flyway stuff to be updated to use the new config keys

**Pull request merged**